stellar-dotnet-sdk/responses/Link.cs

```diff
@@ -14,7 +14,7 @@ public Link(string href)
         Href = href;
     }
 
-    [JsonProperty(PropertyName = "href")] public string Href { get; set; }
+    [JsonProperty(PropertyName = "href")] public string Href { get; }

```

Comment:
Should this not be init instead of a removed set? Like you did in the asset response.

Reply:
Something with those properties from the response classes are not working as I thought. Most of the time it just works without the get, but for some (even in the same class with the working one) it doesn't. 
I will have to take a deeper look some time.